changelogs/unreleased/4613-dependabot.yml

```diff
@@ -0,0 +1,6 @@
+change-type: patch
```

Comment:
why is this one in there?

Reply:
the merge tool will block the merge if there are multiple yml files :)

Reply:
I made pull request from master, and something had to happen in between, as it's treated as something new, I made pull on master and there is that file on master

Reply:
yes, but if you pulled in master, this file should already be present and thus, not be in the diff 🤔

Reply:
I am aware, but this file exist on master already https://github.com/inmanta/web-console/blob/master/changelogs/unreleased/4613-dependabot.yml